OpenShift/helm/nexus-iq/values.yaml

```diff
@@ -4,8 +4,8 @@
 
 iq:
   name: nxiq
-  imageName: registry.connect.redhat.com/sonatype/nexus-iq-server
-  imageTag: 1.85.0-01-ubi
+  imageName: sonatype/nexus-iq-server
```

Comment:
I think this should remain as RH

Reply:
could you elaborate on why the RH one is better? the docker.io image is lower-friction to install, since you don't need to do the whole authentication/key thing that red hat requires.

Reply:
If we do switch it, it would be good to reflect that in the readme, since i think there was specific instructions around defaults (RH) and alternate (docker.io) images.

Reply:
If you check the email from Neil Hartman on Jan 28th, he wrote:

> While continuing our efforts to build a simple helm operator based off the stable helm chart I came across some issues. The main image the helm chart is pulling is the docker image for nexus, not your certified container image that you have hosted in the RHCC, therefore you would need to change that. There are also a few pieces (proxy, and backup) that call images that are hosted on quay. So if you were to certify this operator as is you would need to certify those proxy and backup images as well and reference those certified images in place of what is already being called. 

There were a few back & forth comments later, but my understanding is that using the RHCC image is a requirement of certification.